bom/application/pom.xml

```diff
@@ -4424,7 +4424,13 @@
                 <version>${graal-sdk.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.graalvm.js</groupId>
+                <groupId>org.graalvm.polyglot</groupId>
+                <artifactId>js-community</artifactId>
+                <version>${graal-sdk.version}</version>
+            </dependency>
+            <!-- Same but for Gradle Enterprise -->
```

Comment:
`s/Gradle/Graal VM/`

Reply:
Raaah, I have been working with Gradle Enterprise/Develocity for too long...